.github/actions/spelling/excludes.txt

```diff
@@ -12,15 +12,20 @@ ignore$
 \.npmrc$
 \.png$
 \.jpg$
-\.ttf
+\.ttf$
 \.sav$
 \.sum$
 \.xlf$
+\.svg$
+\.ico$
+\.zip$
+\.gif$
 ^\.github/actions/spelling
 _test\.go$
 ^examples/support/observability/config/
 ^examples/support/observability/assets/
 ^test/prometheus/
+^docs/config/
```

Comment:
why did you exclude this?

Reply:
Because there we will have a lot of YAML keys that I don't think we should maintain for spelling.

Reply:
but there's even text in there that will be rendered directly into the page. e.g. this [here]https://github.com/keptn/lifecycle-toolkit/pull/2460/files#diff-55f3e0185f5274b73ff955662c67559cbe1635aac8a0863dbb4cd871b5071f93R35-R53)
maybe we can make this more granular?

Reply:
valid argument 👍 I'll remove this folder from being excluded